roles/openstack-meta/defaults/main.yml

```diff
@@ -445,3 +445,10 @@ openstack_meta:
           config_dirs: /etc/heat
           config_files: /etc/heat/heat.conf
           restart: on-failure
+  apache:
+    ubuntu:
```

Comment:
if you're adding these here, you should update apache role to not set facts and use them so that we're not doing the same work twice and causing confusion.